### PR TITLE
New Crowdin updates

### LINKED ISSUE #3499: Use Submodule for Avalonia language files
My suggestion is: Create a Submodule for standalone translation will reduce the coding development from approve each PR frequently.

Or use transflex for better translation experience. (Note, discussion ect.)

### PR DESCRIPTION
**Pull request by AcK77**
This update all of our locale files which are updated from our community, everything isn't translated but it's fine to commit some changes right now.

```
French: 80%	
Chinese Simplified: 99%	
Chinese Traditional: 99%	
German: 97%	
Greek: 82%	
Italian: 100%	
Japanese: 98%	
Korean: 99%
Polish: 100%
Portuguese, Brazilian: 99%		
Russian: 86%
Spanish: 94%
Turkish: 100%
Ukrainian: 100%
```

Ukrainian isn't added in Avalonia code so it can't be used right now, will be done in another PR later.

Closes #3499

### CODE REVIEW

**Review by marysaka — APPROVED** [review]

**Review by SiongSng — APPROVED** [review]

**Comment by AcK77** [comment]
Should be good for review now

File: Ryujinx.Ava/Assets/Locales/de_DE.json [hunk 1]
```diff
@@ -81,7 +81,7 @@
   "SettingsTabGeneralRemove": "Entfernen",
   "SettingsTabSystem": "System",
   "SettingsTabSystemCore": "Kern",
-  "SettingsTabSystemSystemRegion": "System Region:",
+  "SettingsTabSystemSystemRegion": "Systemversion:",
```
Comment (gdkchan) [inline Ryujinx.Ava/Assets/Locales/de_DE.json:84]: This looks odd but I don't speak Deutsch.

File: Ryujinx.Ava/Assets/Locales/de_DE.json [hunk 2]
```diff
@@ -129,7 +129,7 @@
   "SettingsTabGraphicsAnisotropicFiltering2x": "2x",
   "SettingsTabGraphicsAnisotropicFiltering4x": "4x",
   "SettingsTabGraphicsAnisotropicFiltering8x": "8x",
-  "SettingsTabGraphicsAnisotropicFiltering16x": "16x",
+  "SettingsTabGraphicsAnisotropicFiltering16x": "16X",
```
Comment (gdkchan) [inline Ryujinx.Ava/Assets/Locales/de_DE.json:132]: Not sure why X is upper case here.

**Review by gdkchan — APPROVED** [review]

File: Ryujinx.Ava/Assets/Locales/de_DE.json (hunk 1 above)
Comment (GamerzHell9137) [reply Ryujinx.Ava/Assets/Locales/de_DE.json:84]: I think it should be Systemregion.

Comment (AcK77) [reply Ryujinx.Ava/Assets/Locales/de_DE.json:84]: Then it should be changed on crowdin

**Pull request merged**